code/iaas/model/src/main/java/io/cattle/platform/core/addon/IpPortPublicEndpoint.java

```diff
@@ -0,0 +1,63 @@
+package io.cattle.platform.core.addon;
+
+import io.github.ibuildthecloud.gdapi.annotation.Type;
+
+@Type(list = false)
+public class IpPortPublicEndpoint {
```

Comment:
Can we change this to PublicEndpoint, as the type ipPortPublicEndpoint will be in the API, ipPort seems unneeded